jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy

```diff
@@ -53,6 +53,7 @@ def pack = { version, os, arch ->
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/grafana/blackbox_exporter.json || true; \
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/grafana/node.json || true; \
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/grafana/kafka.json || true; \
+    wget -qnc https://raw.githubusercontent.com/pingcap/tiproxy/main/pkg/metrics/grafana/tiproxy_summary.json || true; \
```

Comment:
we are now not using this script, edit https://github.com/pingcap/monitoring/blob/master/monitoring.yaml please

Reply:
No need to make change to this file.